student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom)
+    super('unknown', nil, parent_permission: true)
+    @classrom = classroom
+  end
```

Comment:
- [ ] Kindly build your constructor well. You should require age, name and parent_permission and they should be well parsed into the `super` method.

  ```
  def initialize(classroom, age, name = 'unknown', parent_permission: true)
    super(age, name, parent_permission: parent_permission)
    @classroom = classroom
  end
  ```